tests/publiccloud/create_aistack_env.pm

```diff
@@ -0,0 +1,247 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Basic aistack test
+
+# Summary: This test performs the following actions
+#  - Create a VM in EC2 using SLE-Micro-BYOS later than 6.0 version
+#  - Install the required dependencies to install the aistack helm chart
+#  - Test access to OpenWebUI and run integration tests with Ollama and MilvusDB
+# Maintainer: Yogalakshmi Arunachalam <yarunachalam@suse.com>
+#
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use serial_terminal;
+use publiccloud::utils;
+use publiccloud::ssh_interactive;
+use transactional;
+use containers::k8s;
+use strict;
+use warnings;
+use utils;
+use publiccloud::utils;
+use transactional qw(process_reboot trup_install trup_shell);
+use File::Basename;
+use version_utils;
+use Data::Dumper;
+
+sub install_dependency_package {
+    my ($instance) = @_;
+    my $rke2_url = get_var('RKE2_URL');
+    my $kubectl_url = get_var('KUBECTL_URL');
+    my $helm_url = get_var('HELM_URL');
+
+    record_info('Dep pkg install');
+    trup_call("pkg install curl git docker");
+    process_reboot(trigger => 1);
+    systemctl("enable docker");
```

Comment:
Please add here a comment to document the block, like:
`# docker activation section`